project_iteration/static/src/search_input_many2many.js

```diff
@@ -0,0 +1,60 @@
+/*
+    © 2018 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL.html).
+
+    This javascript module allows to display the name of the project in the search bar of tasks.
+
+    Instead of filtering by a many2one field (project_id) like in vanilla Odoo,
+    the view is filtered by a many2many field (parent_project_ids).
+
+    This allows to display the tasks below a parent projects and its iterations.
+
+    Without this module, the ID of the project is displayed in the search bar facet
+    instead of its display name.
+*/
+odoo.define("web_search_input_many2many.search_view_field", function(require) {
+"use strict";
+
+var searchInputs = require("web.search_inputs");
+
+searchInputs.Field.include({
+    facet_for(value){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'facet_for' is not in camel case. (camelcase)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3300066)